scripts/run-verus.sh

```
@@ -5,14 +5,44 @@ SCRIPT_DIR=$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
+                "${REPO_ROOT}/verus/decomposition_cosine_threshold.rs" \
+                "${REPO_ROOT}/verus/decomposition_vector_algebra.rs" \
+                "${REPO_ROOT}/verus/clone_detector_lsh_config.rs"
+            ;;
+        *)
+            printf 'unknown Verus proof group: %s\n' "$1" >&2
+            exit 1
+            ;;
+    esac
+}
+
+PROOF_FILES=($(proof_files_for_group all))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)